test/test_torch.py

```diff
@@ -8438,7 +8439,8 @@ def dims_full_for_fn():
             r1 = fntorch(t0_full, t1, t2)
             self.assertEqual(r0, r1)
 
-    def test_broadcast_batched_matmul(self, device):
+    @tf32_on_and_off(atol_=0.001, rtol_=0.001)
```

Comment:
Why do `atol` and `rtol` have underscores after their names?

Reply:
fixed